src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -576,21 +576,21 @@ def test_sql_db_read_replica_mgmt(self, resource_group, resource_group_location,
                      JMESPathCheck('edition', edition),
                      JMESPathCheck('sku.tier', edition),
                      JMESPathCheck('readScale', 'Enabled'),
-                     JMESPathCheck('readReplicaCount', '1')])
+                     JMESPathCheck('readReplicaCount', 'None')])
```

Comment:
Is there any breaking change in server side?
Typically although we upgrade the api version, we need to make sure there is no breaking change in response.